test/sendmsg_iov_clean.c

```diff
@@ -66,17 +66,14 @@ static void *thread_fn(void *__data)
 	return NULL;
 }
 
-static int queue_sends(struct io_uring *ring, int send_fd, struct thread_data *td)
+static int queue_sends(struct io_uring *ring, int send_fd, struct thread_data *td, struct msghdr* msghdr)
 {
 	struct io_uring_sqe *sqe;
 	char buf[64];
-	struct iovec iovs[IOVS];
-	struct msghdr msghdr = {
-		.msg_iov = iovs,
-		.msg_iovlen = 1,
-	};
 	int i, ret, sbuf;
 
+	struct iovec* iovs = msghdr->msg_iov;
```

Comment:
```struct iovec *iovs = msghdr->msg_iov;```

please, to follow the style. Also don't leave two empty lines between this and the other variable declarations.